Documentation/platforms/arm/imxrt/index.rst

```diff
@@ -0,0 +1,191 @@
+ENC         Yes
+ENET        Yes
+FlexIO      No
+GPIO        Yes
+I2S         Yes
+PWM         No
+SAI         No
+SPDIF       No
+SPI         Yes
+UART        Yes
+USB         Yes
+==========  =======
+
+ACMP
+----
+
+The circout for comparing two analog input voltages designed to operate across the full
+range of supply voltage (rail-to-rail operation).
+
+ADC
+---
+
+ADC driver with the successive approximation analog/digital convertor. The lower-half of
+this driver is initialize by calling :c:func:`imxrt_adcinitialize`.
+
+CAN
+---
+
+FlexCAN driver is supported in MCUs i.MX RT1020 and higher. i.MX RT106x have both classical
+CAN and also one CAN FD while i.MX RT1170 have 3 CAN FD peripherals. FlexCAN driver in imxrt
+works beyond SocketCAN driver layout. The lower-half of this driver is initialize by
+calling :c:func:`imxrt_cannitialize()`.
+
+There is an booting option that automatically provides initialization of network interface
+in the early stages of booting and therefore calling :c:func:`imxrt_cannitialize()` via
+board specific logic is not neccessary. This however works only when there is only one
+interface in the chip. For running more interfaces (like CAN and Ethernet), network late
+initialization must be turn on by CONFIG_NETDEV_LATEINIT and board specific logic must call
+lower-half part of drivers.
+
+CSI
+---
+
+CMOS Sensor interface which enables the chip to connect directly to external CMOS image sensors.
+
+DAC
+---
+
+Digital/analog convertor for external signal is only supported in i.MX RT1170 MCU. It is 12 bit
+lower power, general purpose DAC.
+
+eLCDIF
+------
+
+The enhanced Liquid Crystal Display interface (LCDIF) is a general purpose display controller.
+
+ENC
+---
+
+The enhanced quadrature encoder/decoder module supported in i.MX RT1015 and higher. The lower-half
+of this driver is initialize by calling :c:func:`imxrt_qeinitialize`.
+
+ENET
+----
+
+Ethernet driver supported in i.MX RT1020 and higher. The lower-half of this driver is initialize
+by calling :c:func:`imxrt_netnitialize`.
+
+There is an booting option that automatically provides initialization of network interface
+in the early stages of booting and therefore calling :c:func:`imxrt_cannitialize()` via
+board specific logic is not neccessary. This however works only when there is only one
+interface in the chip. For running more interfaces (like CAN and Ethernet), network late
+initialization must be turn on by CONFIG_NETDEV_LATEINIT and board specific logic must call
+lower-half part of drivers.
+
+FlexIO
+------
+
+A configurable module providing a range of functionality like emulation of a variety of
+serial/parallel communication protocols, flexible 16-bit timers or programmable logic blocks.
+This module is supported in i.MX RT1010 and higher.
+
+GPIO
+----
+
+Pins can be configured using ``imxrt_config_gpio()`` function. Writing to pins is
+done by ``imxrt_gpio_write()`` function and reading is done by ``imxrt_gpio_read()``.
```

Comment:
Use :c:func: 
Not really critical now but it is good to have when we eventully add proper documentation for all these functions.

Reply:
Thanks. I forgot it there back then.